core/src/display_object/edit_text.rs

```diff
@@ -351,45 +280,16 @@ impl<'gc> EditText<'gc> {
         width: f64,
         height: f64,
     ) -> Self {
-        let swf_tag = swf::EditText {
-            id: 0, //TODO: Dynamic text fields don't have a character ID?
-            bounds: swf::Rectangle {
+        let swf_tag = swf::EditText::new()
+            .with_font_id(0, Twips::from_pixels(12.0))
```

Comment:
`Twips::from_pixels_i32` can be used instead. It should be cheaper and it's even a `const fn`.